llvm/lib/Transforms/ObjCARC/ObjCARCOpts.cpp

```diff
@@ -615,15 +617,15 @@ ObjCARCOpt::OptimizeRetainRVCall(Function &F, Instruction *RetainRV) {
       while (IsNoopInstruction(&*I))
         ++I;
       if (&*I == RetainRV)
-        return false;
```

Comment:
Can you make changes to `OptimizeRetainRVCall` in a separate patch? The function always returns `false`, so it'll be a NFC patch.